### DISCUSSION TITLE
Did you discontinue the Code Search page?

### DISCUSSION THREAD

**Discussion by kblestarge** [community/community#10916] [Other Feature Feedback, Questions, & Ideas] [answered]
I get redirected back to github.com when I try to visit the code search page: https://cs.github.com/
Did this page get discontinued? Or did the functionality get moved somewhere?

**Comment by heyajulia** [comment]
You can only use Code Search if you are logged in and enrolled in the beta.

**Comment by colinwm** [comment, marked as answer]
You can still sign up for the technology preview here: https://github.com/features/code-search/signup
